### DISCUSSION TITLE
Friction 0.9.5

### DISCUSSION THREAD

**Discussion by rodlie** [friction2d/friction#112] [Announcements]
Introducing Friction 0.9.5. This release includes several fixes and some new features. See **[release notes](https://friction.graphics/releases/friction-095.html)** for more information.

Happy holidays🎅

https://github.com/friction2d/friction/assets/34516798/98864b3c-d322-4947-87b0-294f0eb7d006

## Checksums *(sha256)*

```
be6c1c9c2636663435b3d0a01311f9a322dae3dc3167ae9a5628f7090e372e0c  friction-0.9.5-1.x86_64.rpm
fc769bd32d969fe519ddf63c0e1406482abd5ef6504606881392b0d6070cd77f  friction-0.9.5-linux-X11-x86_64.tar.xz
a5b673a0eab1c8b723fccb39e8f52f1a9f0e8ab8092017b6f730ebf11b33f0be  friction-0.9.5-portable-win64.7z
e555883f2d035d75431bca895e942e4700065eb881bfa4ed56fcc99dd547e299  friction-0.9.5-setup-win64.exe
126e588b443a2ff45d1ca7336f3c51e89a0faac7b1f2ac4a9cb26f1752dcaf27  friction-0.9.5-ubuntu22.04.deb
ad05c3e8bd47328e88174007562899454c1412499d6fd3e7bcff08cc961919de  friction-0.9.5-ubuntu23.04.deb
2c968b198b361bbb36da5c13948fa5188a5f4317db899e8958e1ae7f2213b59c  friction-0.9.5-ubuntu23.10.deb
1341b121f600f718f4bb992a2249cc0f246b256f9087f866824fb3e3c0983c43  friction-0.9.5-X11-x86_64.AppImage
d5f7871fe488331a9b71193bcf7ac764cf858e79bcc7e0d6ea2a90b94669e1b5  friction-0.9.5-X11-x86_64.flatpak
```

<hr /><em>This discussion was created from the release <a href='https://github.com/friction2d/friction/releases/tag/v0.9.5'>Friction 0.9.5</a>.</em>

**Comment by dadoprom** [comment]
It looks amazing... thank you :)

**Comment by bikass** [comment]
Thank you for picking up and continuing this project. All the best! and looking forward for further development.
